### PR TITLE
Fix the setup steps for localhost

### LINKED ISSUE #564: Tutorial for excel insert file doesn't work
**URL of sample**

[_URL linking to the sample that has the bug._
](https://github.com/OfficeDev/Office-Add-in-samples/tree/1f869ac66cb6a140e628fef263c4ada71ce8834c/Samples/excel-insert-file)

**Describe the bug**

I follow the steps for localhost testing but it doesn't load the addin.

**To Reproduce**

Open the index.js file.

Edit line 4 to refer to the localhost:3000 endpoint as shown in the following code.

const dataSourceUrl = "https://localhost:3000";
Save the file.

You need http-server to run the local web server. If you haven't installed http-server, you can do this with the following command:

npm install --global http-server
Use a tool such as openssl to generate a self-signed certificate that you can use for the web server. Move the cert.pem and key.pem files to the webworker-customfunction folder for this sample.

From a command prompt, go to the web-worker folder and run the following command:

http-server -S --cors . -p 3000
To reroute to localhost run office-addin-https-reverse-proxy. If you haven't installed this proxy, you can do it with the following command:

npm install --global office-addin-https-reverse-proxy
To reroute run the following in another command prompt:

office-addin-https-reverse-proxy --url http://localhost:3000
Follow the steps in [Run the sample](https://github.com/OfficeDev/PnP-OfficeAddins/tree/main/Samples/excel-keyboard-shortcuts#run-the-sample), but upload the manifest-localhost.xml file for step 6.
**Expected behavior**

_A clear and concise description of what you expected to happen._

**Environment**
- Platform [PC desktop, Mac, iOS, Office Online]: Mac Office Online
- Host [Excel, Word, PowerPoint, etc.]: Excel
- Browser (if using Office Online): Chrome

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | yes                               |
| New feature?    | no                              |
| New sample?     | no                               |
| Related issues? | fixes #564 |

## What's in this Pull Request?

Fixes #564.

### CODE REVIEW

File: Samples/excel-insert-file/README.md [hunk 1]
```diff
@@ -58,14 +58,14 @@ To run the sample you just need to sideload the manifest. The add-in web files a
 1. Download the **manifest.xml** and **SalesTemplate.xlsx** files from this sample to a folder on your computer.
```
Comment (AlexJerabek) [inline Samples/excel-insert-file/README.md:58]: Don't forget to update the version history date.

File: Samples/excel-insert-file/README.md [hunk 2]
```diff
@@ -58,14 +58,14 @@ To run the sample you just need to sideload the manifest. The add-in web files a
...
 1. Open [Office on the web](https://www.office.com/).
 1. Choose **Excel**, and then open a new document.
-1. Select the **Insert** tab, and choose **Office Add-ins**.
+1. Select the **Insert** tab, and choose **Add-ins**.
```
Comment (AlexJerabek) [inline Samples/excel-insert-file/README.md:61]: I don't have Add-ins on the Insert tab anymore. Just the Home tab. Not sure if that's the default or a flighting thing.

File: Samples/excel-insert-file/README.md [hunk 3]
```diff
@@ -58,14 +58,14 @@ To run the sample you just need to sideload the manifest. The add-in web files a
...
 
-    ![The Office Add-ins dialog with a drop-down in the upper right reading "Manage my add-ins" and a drop-down below it with the option "Upload My Add-in"](../../Samples/images/office-add-ins-my-account.png)
+    ![The Office Add-ins dialog with a drop-down in the upper right reading "Manage my add-ins" and a drop-down below it with the option "Upload My Add-in"](../images/office-add-ins-my-account.png)
 
 1. Browse to the add-in manifest file, and then select **Upload**.
 
-    ![The upload add-in dialog with buttons for browse, upload, and cancel.](../../Samples/images/upload-add-in.png)
+    ![The upload add-in dialog with buttons for browse, upload, and cancel.](../images/upload-add-in.png)
 
 1. Verify that the add-in loaded successfully. You'll see a **PnP Insert Excel file** button on the **Home** tab.
```
Comment (AlexJerabek) [inline Samples/excel-insert-file/README.md:70]: Is it okay this still has the PnP branding?

**Review by AlexJerabek — APPROVED** [review]
Approving, with a couple questions that don't pertain to this change.

File: Samples/excel-insert-file/README.md (hunk 2 above)
Comment (davidchesnut) [reply Samples/excel-insert-file/README.md:61]: Probably a flighting thing.

File: Samples/excel-insert-file/README.md (hunk 3 above)
Comment (davidchesnut) [reply Samples/excel-insert-file/README.md:70]: Yeah, it's not a high priority as the samples are technically part of the PnP program, but something I would like to clean up eventually. I'll create a separate work item as this is not the only sample with the branding.

File: Samples/excel-insert-file/README.md (hunk 1 above)
Comment (davidchesnut) [reply Samples/excel-insert-file/README.md:59]: thanks fixed!

**Pull request merged**